apps/redi-talent-pool/src/components/templates/LoggedIn.tsx

```diff
@@ -25,6 +26,7 @@ const LoggedIn = ({ children, hideNavigation }: Props) => {
     myTpData.data?.tpCurrentUserDataGet?.tpJobseekerDirectoryEntry
 
   const isBusy = useIsBusy()
+  const location = useLocation()
```

Comment:
Why is this added in this file? Do we use `location` somewhere in this file?

Reply:
@helloanil, yes, here:
<img width="580" alt="image" src="https://github.com/user-attachments/assets/c3c85519-1a5f-4fc0-89a6-27bc5116b923">